studioclass-frontend/src/index.js

```diff
@@ -19,11 +19,13 @@ const authLink = setContext((_, { headers }) => {
   }
 });
 
-// ws://localhost:4000/graphql/subscriptions
-// wss://studioclass.herokuapp.com/graphql/subscriptions
+const wsUri = process.env.NODE_ENV === 'production'
+  ? 'wss://studioclass.herokuapp.com/graphql/subscriptions'
+  : 'ws://localhost:4000/graphql/subscriptions'
```

Comment:
Same thing with URLs.

Storing these constants in env vars would also remove the need for a ternary operator here.